lib/apiservers/engine/backends/container.go

```diff
@@ -57,6 +57,32 @@ import (
 	"github.com/vmware/vic/pkg/uid"
 )
 
+// valid filters as of docker commit 49bf474
+var acceptedPsFilterTags = map[string]bool{
```

Comment:
For maps like this, I would use map[string]struct{}
struct{} doesn't take any space in memory.
For lookup purposes:
```
if _, ok := map[val]; ok {
    ...
}
```